contracts/src/v0.7/dev/KeeperRegistryDev.sol

```diff
@@ -157,6 +171,7 @@ contract KeeperRegistryDev is
    * @param fallbackLinkPrice LINK price used if the LINK price feed is stale
    */
   constructor(
+    uint256 chainID,
```

Comment:
with 0.8 we should be able to use block.chainid (https://docs.soliditylang.org/en/v0.8.0/units-and-global-variables.html)

Reply:
awesome, solid improvement